target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -206,6 +206,9 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:wan" "3:lan" "4:lan"
 		;;
+	dlink,covr-c1200-a1)
+		ucidef_add_switch "switch0" "0@eth0" "1:wan" "2:lan"
```

Comment:
Please use two-line formatting like all the other blocks (although it's not strictly required due to line length)